packages/schemas/alterations/next-1682046821-update-hooks.ts

```diff
@@ -0,0 +1,124 @@
+import { generateStandardId } from '@logto/shared';
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+enum HookEvent {
+  PostRegister = 'PostRegister',
+  PostSignIn = 'PostSignIn',
+  PostResetPassword = 'PostResetPassword',
+}
+
+type HookConfig = {
+  url: string;
+  headers?: Record<string, string>;
+  retries?: number;
+  signingKey?: string;
```

Comment:
since it's an auto-generated field, why don't we just use it as a database field instead of putting it in a JSONB? then you need `hookConfigGuard.omit()` and `Omit<>` and even some confusion in Swagger (different input and output JSON types)..

Reply:
Updated.

Reply:
so why it is still here? i assume it should be a prop of `Hook`